### PR TITLE
Updating of Python Raster Types

### PR DESCRIPTION
**Pull request by abhiataero**
Below Raster types have been updated
- TripleSat
- SuperView
- Datacube-Landsat
- Geoscience
- Geoscience-Landsat
- Geoscience-Sentinel2
- Datacube-Sentinel

### CODE REVIEW

File: types/DataCube-Landsat/DataCube-Landsat.py [hunk 1]
```diff
@@ -0,0 +1,726 @@
...
+                                            'wavelengthMin': 0.0,
+                                            'wavelengthMax': 255.0,
+                                            'datasetTag': 'QA'
+                                        },
+                                        {
+                                            'bandName': 'coastal_aerosol',
+                                            'bandIndex': 8,
+                                            'wavelengthMin': 435.0,
+                                            'wavelengthMax': 451.0,
+                                            'datasetTag': 'QA'
```
Comment (amitjain27) [inline types/DataCube-Landsat/DataCube-Landsat.py]: Is this supposed to be a QA band?

File: types/DataCube-Landsat/DataCube-Landsat.py [hunk 2]
```diff
@@ -0,0 +1,726 @@
...
+
+class Utilities():
+
+    def readYaml(self,path):        #to read the yaml file located locally.
+        try:
+            with open(path, 'r') as q:
+                try:
+                    doc = (yaml.load(q))
+                except yaml.YAMLError as exc:
+                    print(exc)
```
Comment (amitjain27) [inline types/DataCube-Landsat/DataCube-Landsat.py]: We shouldnt be printing exceptions - these will unnecessarily pop up as errors for unrelated raster type identification process.

File: types/DataCube-Landsat/DataCube-Landsat.py (hunk 1 above)
Comment (abhiataero) [reply types/DataCube-Landsat/DataCube-Landsat.py]: this has been fixed

File: types/DataCube-Landsat/DataCube-Landsat.py (hunk 2 above)
Comment (abhiataero) [reply types/DataCube-Landsat/DataCube-Landsat.py]: this has been taken care of

**Pull request closed without merging**